build/publish/PublishNupkgToTransportFeed.targets

```diff
@@ -6,6 +6,12 @@
   </ItemGroup>
 
   <PropertyGroup>
+    <BuildTasksFeedDll>$(NuGetPackagesDir)\microsoft.dotnet.build.tasks.feed\$(BuildTasksFeedDllVersion)\build\netstandard1.5\Microsoft.DotNet.Build.Tasks.Feed.dll</BuildTasksFeedDll>
```

Comment:
Why did we switch to look at the nuget cache versus the dotnet-cli-build output?